File: URI/1861.py
dicionario = {}
List_deads = [0]
x = raw_input()
index = 0
while(x):
	ass,dead = x.split(" ")
	
	if(ass in dicionario.keys()):
		dicionario[ass] += 1
	else:
		dicionario[ass] = 1	
	
	List_deads.insert(index,dead)
	index = index +1
	
	x = raw_input()

for i in List_deads:
	if(i in dicionario.keys()):
		del dicionario[i]	

print("HALL OF MURDERERS")
print(dicionario)
dicionario.keys().sort()
print(type(dicionario.keys()))

File: URI/1861.cpp
#include <iostream>
#include <string>
#include <map>
using namespace std;

int main(){
	map<string,int> killer;
	map<string,int> dead;
	map<string, int> :: iterator it;
	
	string k,d;
	
	
	
	while(cin >> k >> d){
		killer[k]++;
		dead[d]++;
		if(k == " ")break;
	}
	
    cout << "HALL OF MURDERERS" << endl;
	for(it = killer.begin(); it != killer.end(); it++){
		string ass = it->first;
		if(dead.count(ass) == 0){
			cout << it->first << " " << it->second << endl;
		}
		
	}
	
	return 0;

}
